src/include/daos_srv/pool.h

```diff
@@ -50,6 +50,7 @@ struct ds_pool {
 	ABT_rwlock		sp_lock;
 	struct pool_map	       *sp_map;
 	uint32_t		sp_map_version;	/* temporary */
+	uint64_t		sp_reclaim;
```

Comment:
Why not uint32_t if it is only for enum values?

Reply:
Because it's defined as uint64_t in both pool_iv_prop & daos_prop_entry, we could change it to uint32_t in a separate patch.

Reply:
I see. Thanks.